withdraw-by-check/withdraw-by-check-json-contract.md

```diff
@@ -79,71 +124,47 @@ If the request is successful, the poweron should respond with:
 
-### Errors
-If the request is not successful, the poweron should respond with one of the following errors:
-
-If there was an issue opening or reading the configuration Letter file
+**When error condition is encountered:**
 ```json
 {
-  "errorCode": 500,
-  "loggingErrorMessage": "Error [opening/reading] Letter file: +[error returned]"
+  "errorCode": 999,
+  "loggingErrorMessage": "error message detail",
+  "requested": "123456.00",
+  "minWdAmount": "123456.00",
+  "minWdAmount": "123456.00"
 }
 ```
-
-If the request is not successful due to insufficient funds, the poweron should respond with the
-following and the client may try request again with a lesser amount:
-
-```json
-{
-  "errorCode": 501,
-  "loggingErrorMessage": "Amount req. ###,##9.99 exceeds avail. ###,##9.99"
-}
-```
-In the case of insufficient funds, the client may try request again with a lesser amount.
-
-Should the program determine that the member's address is insufficient to mail a check, the
-response would be as follows:
-
-```json
-{
-  "errorCode": 502,
-  "loggingErrorMessage": "Invalid address"
-}
-```
-
-If the request is not successful due to an invalid account, loan or share, the poweron would respond with:
-
-```json
-{
-  "errorCode": 503,
-  "loggingErrorMessage": "[Account Not Found/Account Warning Found/Share Not Found/Loan Not Found/Share Warning Found/Loan Warning Found]"
-}
-```
-
-If the request is not successful due to Reg D Limits, the response will be:
-```json
-{
-  "errorCode": 504,
-  "loggingErrorMessage": "TRANPERFORM Error: Reg D Limit"
-}
-```
-
-If the member attempts to make a WD from a cross account, the poweron will respond with:.
-
-```json
-{
-  "errorCode": 505,
-  "loggingErrorMessage": "Cross Account WD Attempted"
-}
-```
-
-### Error codes
-| Code   | Description              |
-|--------|--------------------------|
-| 500    | CFG file error           |
-| 501    | Insufficient Funds       |
-| 502    | Invalid address          |
-| 503    | Invalid Acct/loan/share  |
-| 504    | Reg D Limit              |
-| 505    | X-Acct Error             |
+**Error detail:**
+Unless otherwise noted, all values will be passed as double-quote encapsulated character values
+ - errorCode: error code generated (numeric)
+ - loggingErrorMessage: Error message specific to the error code
+ - requested: The amount of the WD that was requested
+ - minWdAmount: the minimum WD amount (from parameter settings)
+ - maxWdAmount: the maximum WD amount (from parameter settings)
+
+**Error Code detail:**
+|Error Code| Logging Error Message Detail|
+|-|-|
+|500|Error Opening Letterfile BANNO.CHECK.WITHDRAW.V1.CFG: *[system generated error message]*
+|501|Target Share/Loan available balance<=$0.00
+|501|Amount requested exceeds available
+|502|Invalid address|
+|503|Account not found|
+|503|Account warning found|
+|503|Target Share/Loan not found|
+|503|Share/Loan invalid type|
+|503|Share/Loan warning found|
+|506|Amount requested out of bounds|
```

Comment:
Error code 504 "Reg D Limit" is missing
Error code 505 "Cross Account WD Attempted" is missing
I think those need to be added back

Side note: Error Code 500 can also be "TRANPERFORM Error:" not sure if listing that in the json contract is important